src/SaveNicknameToDatabase.php

```diff
@@ -16,7 +16,9 @@ public function handle(Saving $event)
         $attributes = Arr::get($data, 'attributes', []);
```

Comment:
Do we check on the backend to ensure the display name driver is nicknames? I can't see it in this file, but it might be done elsewhere.

Reply:
No, but I don't think we need to. We don't show the controls on the default frontend to avoid confusion, but its conceivable that someone might want to edit nicknames via REST api regardless of which display driver is enabled.